docs/pipelines.md

```diff
@@ -1,4 +1,11 @@
 # Pipelines
 
-* Deploy to Gov PaaS - This is a simple pipeline to demonstrate capabilities.  Builds, tests and deploys a simple python application to the PaaS for evaluation in Dev and Test Only.
-* Build and Deploy Forms - This builds the form runner (using `Dockerfile`) with the forms contained in `form_jsons` and deploys to dev (and on main, to test as well).
+## Frontend Python application
+Deploy to Gov PaaS (`govcloud.yml`) - Builds, tests and deploys a simple python application to PaaS in Dev on every push, and Test if on main. Not triggerred on updates to the form_jsons folder.
+
+## Form Runner
+Build and Deploy Forms (`build-deploy-forms.yml`) - This builds the form runner (using `Dockerfile`) with the forms contained in `form_jsons` and deploys to dev (and if on `main`, to test as well).
+
+Note that the form runner deployment is only triggered automatically on update of the contents of `form_jsons` in this repo, it does not automatically trigger on an update of the form runner image.
+
+The form runner image is created from the [Digital Form Builder Repo](https://github.com/communitiesuk/digital-form-builder) using the pipeline [DLUHC Build and Publish](https://github.com/communitiesuk/digital-form-builder/actions/workflows/dluhc-build-and-publish.yml). This pushes a new image to https://github.com/communitiesuk/digital-form-builder/pkgs/container/digital-form-builder-dluhc-runner. Any tag from that docker repo can be used in the `Dockerfile` in this repo but if you just want to redeploy the forms with the latest version of the form runner you can manually trigger [Build and Deploy Formst](https://github.com/communitiesuk/funding-service-design-frontend/actions/workflows/build-deploy-forms.yml) in this repo which will just pickup the `latest` tag of the form runner.
```

Comment:
Just to dig a little into this bit. The runner workflow (https://github.com/communitiesuk/digital-form-builder/actions/workflows/dluhc-build-and-publish.yml) pushes to the registry (when tagged). We can also push to the registry in the frontend workflow[ here](https://github.com/communitiesuk/funding-service-design-frontend/blob/main/.github/workflows/build-deploy-forms.yml#:~:text=name%3A%20Build%20and%20push%20docker%20image) , this uses the Dockerfile in this frontend repo. [Here](https://github.com/communitiesuk/funding-service-design-frontend/blob/main/.github/workflows/build-deploy-forms.yml#:~:text=%2D%20name%3A%20Build,COMMIT_SHA%3D%3A%24%7B%7B%20github.sha%20%7D%7D) and [here](https://github.com/communitiesuk/digital-form-builder/blob/main/.github/workflows/dluhc-build-and-publish.yml#:~:text=%2D%20name%3A%20Build%20and,LAST_COMMIT%3D%27%24%7B%7B%20github.sha%20%7D%7D%27)

Reply:
Ah I see.... I will make it clearer :)